MSSQL/OMOPLoader.sql

```diff
@@ -22,108 +22,175 @@
 -- create synonyms to make the code portable - please edit these
 ----------------------------------------------------------------------------------------------------------------------------------------
 
--- Change to your omop database
-use i2b2stub;

```

Comment:
Fine to remove, but see my comment on the instructions at the beginning that should highlight that it needs to be run from the OMOP db.

Reply:
Add the comment on which schema to use back in.